weblab/templates/registration/login.html

```diff
@@ -12,7 +12,7 @@ <h1>Log in</h1>
     {{ form.as_p }}
     <p>
       <button>Log in</button>
-      <a class="button" href="{% url "password_reset" %}">Reset Password</a>
+      <p><a href="{% url "password_reset" %}">Reset Password</a></p>
```

Comment:
Given we're already in a `<p>` here, maybe close that and then start a new one for the link? But otherwise looks fine!